packages/sw-precaching/package.json

```diff
@@ -21,5 +21,9 @@
   "homepage": "https://github.com/GoogleChrome/sw-helpers/tree/master/packages/sw-precaching",
   "main": "build/sw-precaching.min.js",
   "module": "build/sw-precaching.min.mjs",
-  "jsnext:main": "build/sw-precaching.min.mjs"
+  "jsnext:main": "build/sw-precaching.min.mjs",
+  "devDependencies": {
```

Comment:
This test is a smoke test and is just to ensure the precaching route is feasible. I really didn't want to add a build step for routing as it's likely to get messy and un-maintainable since it won't be the same build step as sw-routing. #twocents